esp/esdllib/esdl_transformer2.cpp

```diff
@@ -1558,13 +1561,39 @@ Esdl2Transformer::~Esdl2Transformer()
 
 }
 
+static const char *queryMethodMetaData(IEsdlDefMethod *mthdef, IEsdlMethodInfo *mi, const char *tag)
```

Comment:
Is this correct: this function takes either mthdef or mi because it's called from different locations that will already have a pointer to either one or the other. Furthermore, the mi object actually contains a pointer to the IEsdlDefMethod, so either way we are getting the information from essentially the same place.

Reply:
I'll see if I can simplify it then.

Reply:
Yes that is correct.  I just wanted a single consolidated easy to call function that took either interface type.  They do ultimately get the info from the same place.